src/styles/utile/_colors.scss

```
@@ -0,0 +1,4 @@
+$color-day: #eeeeeeed;
```

It's not a mistake but in my opinion, you do not need separate file for colors, you can put it in variables and gruop maybe . Give you code from documentation:
```suggestion
@use "sass:map";

$theme-colors: (
  "success": #28a745,
  "info": #17a2b8,
  "warning": #ffc107,
);

.alert {
  // Instead of $theme-color-#{warning}
  background-color: map.get($theme-colors, "warning");
}

```

---

i don't see it in documentation. Can you provide me a link?

---

https://sass-lang.com/documentation/variables#advanced-variable-functions